m01_introduction/02_languages.rst

```diff
@@ -46,14 +46,13 @@ A multilingual RST index may look like this:
       Finnish <index_fi>
       English <index_en>
 
-The above index includes a title, maximum tree depth, and link names so that it
-compiles into sensible HTML for non A-plus use.
+The above ``index.rst`` file includes a title, maximum tree depth, and link names so that it compiles into sensible HTML for non A-plus use.
 However, the only A-plus requirements are that the index includes
-a **toctree** that has the option **:caption: Select language**.
+a ``toctree`` that has the option ``:caption: Select language``.
 
-The links are to the the different RST language indexes that must include
+The links are to the different *RST* language indexes that must include
```

Comment:
Better grammar: The links are -> The links point to the different RST language indexes